src/ErrorLogger/styles.js

```diff
@@ -9,25 +9,25 @@ export const ErrorDisplay = styled.div`
     position: fixed;
     max-height: 300px;
     overflow-y: scroll;
-    background-color: ${props => props.displayBackground || '#1E2D53'};
-    box-shadow: ${props => props.displayShadow || '0 -2px 20px 0 rgba(20,31,60,0.65)'};
-    z-index: ${props => props.zIndexInput || 'auto'};
+    background-color: #1E2D53;
+    box-shadow: 0 -2px 20px 0 rgba(20,31,60,0.65);
+    z-index: auto;
```

Comment:
`auto` is the default value for `z-index`, so these can be removed.

Reply:
Thank you for the feedback